hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+If a test raises a "skip" exception from a testing framework, like :func:`pytest:pytest.skip`, we no longer save the input which caused the skip exception to the :ref:`example database <database>`. We assume skip exceptions are an expected outcome of a test, and not something we should try to replay.
```

Comment:
Disagree - we _do_ want to save inputs which cause skip exceptions, so that we waste as little time as possible later on exploring tests that will ultimately be skipped.

so... just the realization fix?

Reply:
ah, this is a good point. I'll revert this then